docs/user_manual/components.md

```diff
@@ -695,15 +695,116 @@ A `fault` has no steady state output.
+| `line_drop_compensation_x` | `double`                                                                                                                                                                                                                                                                                                           | ohm (Ω)  | compensation for voltage drop due to  reactance during transport (see [below](#line-drop-compensation)) |                  &#10060; default `0.0`                  | &#10004; |      `>= 0`       |
+
+The following additional requirements exist on the input parameters.
+
+- The `control_side` supports {py:class}`BranchSide <power_grid_model.enum.BranchSide>` if it regulates a {hoverxreftooltip}`user_manual/components:transformer` and {py:class}`Branch3Side <power_grid_model.enum.Branch3Side>` if it regulates a {hoverxreftooltip}`user_manual/components:Three-Winding Transformer`.
+- The rated voltage at the `tap_side` must be greater than or equal to the rated voltage at the `control_side`.
+- The voltage band must be sufficiently large, i.e., it must be greater than the largest change in voltage due to a change in tap position. This means, that
+  $U_{\text{band}} \geq \frac{\text{tap_size}/u_\text{tap_side}}{\left(1 + \left(\left|\text{tap_min}\right| - \left|\text{tap_nom}\right|\right)\frac{\text{tap_size}}{\text{u_tap_side}}\right)^2}u_{\text{control_side}}$
+- The line drop compensation is small, in the sense that its product with the typical current through the transformer is much smaller (in absolute value) than the smallest change in voltage due to a change in tap position. This means, that
+  $\left|z_{\text{compensation}}\right|\left|I_{\text{node}}\right| \ll \frac{\text{tap_size}/u_\text{tap_side}}{\left(1 + \left(\left|\text{tap_max}\right| - \left|\text{tap_nom}\right|\right)\frac{\text{tap_size}}{\text{u_tap_side}}\right)^2}u_{\text{control_side}}$
+
+These requirements make sure no edge cases with undefined behavior are encountered. Typical real-world power grids already satisfy these requirements and they should therefore not cause any problems.
+
+#### Steady state output
+
+| name      | data type | unit | description          |
+| --------- | --------- | ---- | -------------------- |
+| `tap_pos` | `int8_t`  | -    | optimal tap position |
+
+#### Short circuit output
+
+A `transformer_tap_regulator` has no short circuit output.
+
+#### Electric Model
+
+The transformer tap regulator itself does not have a direct contribution to the grid state.
+Instead, it regulates the tap position of the regulated object until the voltage at the control side is in the specified voltage band:
+
+$$
+   \begin{eqnarray}
+      U_{\text{control}}$ is in the range $\left[U_{\text{set}} - \frac{U_{\text{band}}}{2}, U_{\text{set}} + \frac{U_{\text{band}}}{2}\right]
```

Comment:
"is in the range" should be as text. Formatting is not good now, see: https://power-grid-model--606.org.readthedocs.build/en/606/user_manual/components.html#id40